drivers/bridge/bridge.go

```diff
@@ -416,6 +416,11 @@ func (d *driver) configure(option map[string]interface{}) error {
 		})
 	}
 
+	if config.EnableIPTables && config.EnableIP6Tables && config.EnableUserlandProxy {
```

Comment:
this doesn't look right, trying to understand the negative impact if this snippet was missing

Reply:
Because I only made changes to the dummy proxy the real one has to be disabled for if iptables and ip6tables are enabled. Otherwise we have the listen issue described below because the double call of `allocatePortsInternal`.

Without this lines the userland proxy has to be disabled manually or it is not possible publish any port via `0.0.0.0` or `::` as host address (if ip6tables is enabled).

Also if iptables and ip6tables are enabled I think there is no use for the userland proxy anymore (but here I am not sure).